File: GroupProjectFestivalApp/Presentation/ngApp/controllers/commentController.ts
namespace GroupProjectFestivalApp.Controllers {
    export class CommentController {
        
        public comments;        

        constructor(private $http: ng.IHttpService, private $routeParams) {
            $http.get(`/api/comments`)
                .then((response) => {
                    this.comments = response.data;
                });
        }
    }
}

File: GroupProjectFestivalApp/Presentation/Controllers/CommentController.cs
using GroupProjectFestivalApp.Services;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;

namespace GroupProjectFestivalApp.Presentation.Controllers {
    public class CommentController : ApiController {

        private EventService _eventService;

        public CommentController(EventService eventService) {
            _eventService = eventService;
        }

        [HttpGet]
        [Route("api/event/comments")]
        public IList<CommentDTO> GetComment() {
            return _eventService.GetComment();
        }
    }
}

